src/packages/end-to-end/src/__tests__/basic-mutation.test.ts

```diff
@@ -0,0 +1,27 @@
+import request from 'supertest-graphql';
+import gql from 'graphql-tag';
+
+import { Album } from '../types';
+import { config } from '../config';
+import { resetDatabase } from '../utils';
+
+describe('basic mutation', () => {
+	beforeEach(resetDatabase);
+
+	test('should create an album', async () => {
+		const { data } = await request<{ createAlbum: Album }>(config.baseUrl)
+			.mutate(
+				gql`
+					mutation CreateAlbum($data: AlbumInsertInput!) {
+						createAlbum(data: $data) {
+							id
+						}
+					}
+				`
+			)
+			.variables({ data: { ArtistId: { id: 1 }, Title: 'string' } })
+			.expectNoErrors();
+
+		expect(data?.createAlbum?.id).toBe('348');
```

Comment:
Is this id expected just because we know there are 347 other albums already present? Does this test fail if you run it twice?

Reply:
each time the test is run the database is reset. So that the number is always 348.